app/build.gradle

```diff
@@ -62,8 +62,11 @@ dependencies {
     implementation 'org.jogamp.jogl:jogl-all:2.3.2'
     implementation 'de.hdodenhof:circleimageview:3.0.0'
     implementation 'com.github.PhilJay:MPAndroidChart:v3.1.0'
+    implementation 'com.android.support:recyclerview-v7:28.0.0'
```

Comment:
Can you clean up these dependencies? Move the version cods to root gradle file as done in from line 44~51 and remove duplicates. Also keep similar libraries together (Eg: android libraries goes with android libraries)